--- FAISS load error for user_id=products_index ---
Traceback (most recent call last):
  File "/Users/khaneapple/Documents/freelance/zibochat/llm_agent.py", line 107, in load_user_index
    store = FAISS.load_local(str(d), embedder)
  File "/Users/khaneapple/Documents/freelance/zibochat/.venv/lib/python3.13/site-packages/langchain_community/vectorstores/faiss.py", line 1190, in load_local
    raise ValueError(
    ...<10 lines>...
    )
ValueError: The de-serialization relies loading a pickle file. Pickle files can be modified to deliver a malicious payload that results in execution of arbitrary code on your machine.You will need to set `allow_dangerous_deserialization` to `True` to enable deserialization. If you do this, make sure that you trust the source of the data. For example, if you are loading a file that you created, and know that no one else has modified the file, then this is safe to do. Do not set this to `True` if you are loading a file from an untrusted source (e.g., some random site on the internet.).

